CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing guidelines
+
+Thank you for your interest in the project. We look forward to your
+contribution. In order to make the process as fast and streamlined as possible,
+here is a set of guidelines we recommend you follow.
+
+## Reporting issues
+
+We do our best to ensure bugs don't creep up in our releases, but some may
+still slip through. If you encounter one while using docker-py, please create
+an issue [in the tracker](https://github.com/docker/docker-py/issues/new) with
+the following information:
+
+- docker-py version, docker version and python version
```

Comment:
I think it could make sense to mention OS/distribution and whether docker-py comes from PyPI.